dispatcher/dispatcher.go

```diff
@@ -427,9 +427,19 @@ func (d *IotxDispatcher) HandleBroadcast(ctx context.Context, chainID uint32, pe
 			log.L().Debug("Failed to handle consensus message.", zap.Error(err))
 		}
 	case iotexrpc.MessageType_ACTION:
-		d.dispatchAction(ctx, chainID, message)
+		d.dispatchAction(ctx, chainID, message.(*iotextypes.Action))
+	case iotexrpc.MessageType_ACTIONS:
+		acts := message.(*iotextypes.Actions)
+		for i := range acts.Actions {
+			d.dispatchAction(ctx, chainID, acts.Actions[i])
+		}
 	case iotexrpc.MessageType_BLOCK:
-		d.dispatchBlock(ctx, chainID, peer, message)
+		d.dispatchBlock(ctx, chainID, peer, message.(*iotextypes.Block))
+	case iotexrpc.MessageType_BLOCKS:
```

Comment:
broadcast won't need to handle BLOCKS? each new block will be broadcast immediately, cannot wait to pack them and send in batch